module.combodo-approval-extended.php

```diff
@@ -73,7 +78,28 @@ class ApprovalExtendedInstaller extends ModuleInstallerAPI
 	{
 		public static function BeforeWritingConfig(Config $oConfiguration)
 		{
-			// If you want to override/force some configuration values, do it here
+			if (empty(utils::GetConfig()->GetModuleSetting('combodo-approval-extended', 'targets', null))
+					&& !empty(utils::GetConfig()->GetModuleSetting('combodo-approval-extended', 'target_state', null) )) {
+				// Migration of the old configuration to the new one
+				$newConfiguration = [
+													'UserRequest' =>   [
+														'target_states' => [$oConfiguration->GetModuleSetting('combodo-approval-extended', 'target_state', null) ],
+														'bypass_profiles' =>$oConfiguration->GetModuleSetting('combodo-approval-extended', 'bypass_profiles', 'Service Manager') ,
+														'reuse_previous_answers' => $oConfiguration->GetModuleSetting('combodo-approval-extended', 'reuse_previous_answers', true) ,
+													]
+												];
```

Comment:
This will look much better:
```suggestion
					'UserRequest' =>   [
						'target_states' => [$oConfiguration->GetModuleSetting('combodo-approval-extended', 'target_state', null) ],
						'bypass_profiles' =>$oConfiguration->GetModuleSetting('combodo-approval-extended', 'bypass_profiles', 'Service Manager') ,
						'reuse_previous_answers' => $oConfiguration->GetModuleSetting('combodo-approval-extended', 'reuse_previous_answers', true) ,
						]
					];
```